invenio_records_rest/views.py

```diff
@@ -95,7 +95,10 @@ def elasticsearch_badrequest_error(error):
         for cause_type, handler in handlers.items():
             if cause_type in cause_types:
                 return handler(error)
-        return error
+
+        exception = UnhandledElasticsearchError()
```

Comment:
It seems like the Elasticsearch error is swallowed here thus you won't know exactly what error couldn't be handled.